gen-api-reference/README.md

```diff
@@ -38,3 +38,40 @@ at the root of the repo, and that your GOPATH is set properly. For example:
 1. Run `git diff` to verify the changes.
 
 1. Create a PR to merge your changes.
+
+# Generating the Pipelines API Reference Page
+
+These are instructions for generating a copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/) page from the swagger file. 
+
+## Prerequisites
+* Download the [pipelines](https://github.com/kubeflow/pipelines) repository.
+* Install [npm](https://www.npmjs.com/get-npm).
+
+## Installation
+
+1. Extract the pipelines repository to a local directory.
+
+1. Install [bootprint](https://github.com/bootprint/bootprint-openapi) from the command line.
```

Comment:
Could this step and the next step be combined? For example:

1. Install [bootprint](https://github.com/bootprint/bootprint-openapi) and [html-inline](https://github.com/substack/html-inline) using the following command:
   ```
   npm install -g bootprint
   npm install -g bootprint-openapi
   npm install -g html-inline
   ```

Reply:
Good idea, they've been combined!